File: src/program/rust/fizzbuzz/fizzbuzz.rs
fn main() {
    for n in 1..=50 {
        if n % 3 == 0 && n % 5 == 0 {
            println!("fizz buzz");
        } else if n % 3 == 0 {
            println!("fizz");
        } else if n % 5 == 0 {
            println!("buzz");
        } else {
            println!("{}", n);
        }
    }
}

File: src/program/julia/FizzBuzz/src/FizzBuzz.jl
module FizzBuzz

export main

function fizzbuzzelementstring(n::T) where {T <: Integer}
    n3 = iszero(n % T(3))
    n5 = iszero(n % T(5))
    if n3 && n5
        "fizz buzz"
    elseif n3
        "fizz"
    elseif n5
        "buzz"
    else
        string(n)
    end
end

(printfizzbuzz(n::T) where {T <: Integer}) = foreach(println ∘ fizzbuzzelementstring, one(T):n)

(@main)(::Vector{String}) = printfizzbuzz(30)

end # module FizzBuzz
